source/driver-models/Timer.cpp

```diff
@@ -228,8 +228,12 @@ void Timer::sync(CODAL_TIMESTAMP t)
     // First, update our timestamps.
     currentTimeUs += t;
     overflow += t;
-    currentTime += overflow / 1000;
-    overflow = overflow % 1000;
+
```

Comment:
@mmoskal Can you just double check this change (I think it works out).

Reply:
I think `overflow` used to be 64 bit, for which div/mod are always very slow. On M4 there's div instruction, and I guess mod must be implemented with it. On M0 there's is no div. Maybe add a comment that this is time critical, and does at most one loop iteration per millisecond? ;)